oneanddone/tasks/forms.py

```diff
@@ -50,7 +50,7 @@ class TaskForm(forms.ModelForm):
                 help_text=_('Please use commas to separate your keywords.'),
                 required=False,
                 widget=forms.TextInput(attrs={'class': 'medium-field'})))
-    owner = forms.ModelChoiceField(queryset=User.objects.filter(is_staff=True).order_by('profile__name'))
+    owner = forms.ModelChoiceField(queryset=User.objects.filter(is_staff=True).order_by('profile__name').exclude(profile__name=None))
```

Comment:
This is good, @tessie, but the line is a bit too long. Could you find a way to break it up to be on multiple lines?